packages/module/patternfly-docs/content/extensions/chatbot/examples/demos/Chatbot.md

```diff
@@ -72,22 +72,22 @@ This demo displays a basic chatbot, which includes:
 5. A [`<ChatbotFooter>`](/patternfly-ai/chatbot/ui#footer) with a [`<ChatbotFootNote>`](/patternfly-ai/chatbot/ui#footnote-with-popover) and a `<MessageBar>` that contains the abilities of:
 
 - [Speech to text.](/patternfly-ai/chatbot/ui#message-bar-with-speech-recognition-and-file-attachment)
-- Sending a message to the chatbot.
+- Sending a message to the ChatBot.
 - Receiving a response from a backend AI tool with a loading message state.
 
 6. A [`<ChatbotConversationHistoryNav>`](/patternfly-ai/chatbot/ui#navigation) toggled open and closed by the `<ChatbotHeaderMenu`> in the `<ChatbotHeader>`.
 
-7. A "skip to chatbot" button that allows you to skip to the chatbot content via the [PatternFly skip to content component](/patternfly-ai/chatbot/ui#skip-to-content). To display this button you must tab into the main window.
+7. A "skip to ChatBot" button that allows you to skip to the chatbot content via the [PatternFly skip to content component](/patternfly-ai/chatbot/ui#skip-to-content). To display this button you must tab into the main window.
```

Comment:
The demo code has `Skip to chatbot` - do you want that changed?

Reply:
Oh hmmm good callout! I'm thinking we should actually leave it lowercase how it is now, since it's more of a generic/all-inclusive button and doesn't need to be associated with the "branded" PF ChatBot extension by the time the end users are seeing it. Then, it'll also work regardless of what the specific bot is named. What do you think?